### DISCUSSION TITLE
First trial as a baseline with current parameter values

### DISCUSSION THREAD

**Discussion by lkaino** [optuna/optuna#5821] [Q&A]
I'm using TPESampler and I wanted to run the first trial as a baseline with current parameter values. This is to provide visual feedback in the dashboard in form of reported steps and the final value. It also helps in pruning out the trials with less than baseline result.

I tried implementing this by limiting the search spaces of the first trial, but this resulted in TPESampler disabling multivariate sampling.

Is there another way to achieve this? Should I just run the baseline without any suggested parameters?

**Comment by nzw0301** [comment]
How about https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.Study.html#optuna.study.Study.enqueue_trial?
